frontend/src/pages/settings/index.vue

```diff
@@ -141,10 +141,11 @@ import { version as clientVersion } from '@/../package.json';
 const { t } = useI18n();
 const route = useRoute();
 const remote = useRemote();
-let systemInfo = {} as SystemInfo;
 
 route.meta.title = t('settings.settings');
 
+let systemInfo: SystemInfo = {};
```

Comment:
Why this position change?

Reply:
The position was changed so that it's right above the part of the code where it gets re-assigned. I just thought it was slightly easier to follow. Happy to move it back, but heads up that the line will still have a diff since the type declaration style is changed.

Reply:
@aweebs No problem, just was curious about it. I personally stick to leave all the variables at the top of the component, but I don't have any strong feelings about it either.